demos/stokes/setup.py

```diff
@@ -0,0 +1,80 @@
+from firedrake import *
+from firedrake_adjoint import *
+from pyroteus.metric import *
+from pyroteus.recovery import *
+from opt_adapt.opt import get_state
+
+def initial_mesh(n):
+    """
+    Construction of initial mesh
+    """
+    mesh = Mesh("stokes/stokes_control.msh")
+    return mesh
+
+
+def initial_control(mesh):
+    """
+    The initial of control parameter
+    In this example, control parameter is not in Real space
+    """
+    V = VectorFunctionSpace(mesh, "Lagrange", 2)
+    return Function(V, name="Control")
+
+
+def forward_run(mesh, control, **kwargs):
```

Comment:
Please add an outfile kwarg here, too. See the `turbine` demo for an example of how this works with mixed spaces.